WebApp/autoreduce_webapp/templates/help.html

```diff
@@ -48,108 +48,153 @@ <h2>ISIS Compute</h2>
+                "Add new run variables" below the status panel on the instrument summary page.
+                Clicking either of these links will take you to a form in which you can set the
+                values of variables exposed by that instruments reduce script.
             </p>
         </div>
     </section>
+
     <section class="help-topic panel panel-default" data-topics="invalid variables wrong error value">
         <div class="panel-heading">
             <h4>When changing variables why is value X invalid for variable Y?</h4>
         </div>
         <div class="panel-body">
             <p>
-                Any value entered for a variable must match the type of the default value already in the reduction script. If any value doesn't match you should see a warning message explaining the reason and you will be prevented from submitting new variables until this is resolved.
+                Any value entered for a variable must match the type of the default value already in
+                the reduction script. If any value doesn't match you should see a warning message
+                explaining the reason and you will be prevented from submitting new variables until
+                this is resolved.
             </p>
             {% if support_email %}
-                <p>If you feel a value is incorrectly being marked as invalid please <a href="mailto:{{support_email}}">Contact Us</a>.</p>
+                <p>If you feel a value is incorrectly being marked as invalid please
+                    <a href="mailto:{{support_email}}">Contact Us</a>.</p>
             {% endif %}
         </div>
     </section>
+
     <section class="help-topic panel panel-default" data-topics="reduction data file location directory output reduced">
         <div class="panel-heading">
             <h4>I've modified my reduction scripts, why aren't my new runs using it?</h4>
         </div>
         <div class="panel-body">
             <p>
-                To make sure that reductions coming straight from the instrument use new reduce.py scripts automatically, on the All Jobs page
-                click on one of the instrument name, which brings up the Upcoming Configurations page. Then on the left hand side of this page
-                click on 'edit variables' in the Current Variables section on this page. This brings up a Configure New Jobs page. Here ensure
-                that the 'Track script changes' checkbox is ticked.
+                To make sure that reductions coming straight from the instrument use new <code>reduce.py</code>
+                scripts automatically, you can navigate to the instrument summary page fort a
+                given instrument, and select the "See instrument variables" located on the right
+                in relation ot the status table to view ongoing and upcoming instrument
+                configurations.
+                From here, you can select "edit variables" on the left side of the upcoming
```

Comment:
`"edit variables"` capitalised?

Reply:
Leaving this as is to quote link text exactly including capitalisation (or lack of in this case)